spec/factories/teams.rb

```diff
@@ -11,6 +11,10 @@
       season { Season.current }
     end
 
+    trait :last_season do
+      season { Season.prev }
```

Comment:
I don't understand why, but it doesn't.  If I do it like that ^ or (that^).to_s, it raises a validation error when I reset the db. If I add a :prev_season to the Season factory and create or call that one from the teams factory, it is not recognized (either not registered or unknown constant.) 
???